feat: Updating integrations section (#536)
apify/apify-docs: sources/platform/integrations/slack.md

---
title: Slack

> description: Learn how to integrate your Apify actors with Slack. This article guides you from installation through to automating your whole workflow in Slack.

Reviewer: ```suggestion
description: Learn how to integrate your Apify Actors with Slack. This article guides you from installation through to automating your whole workflow in Slack.
```

sidebar_position: 11.13
slug: /integrations/slack
---

# Slack integration

**Learn how to integrate your Apify actors with Slack. This article guides you from installation through to automating your whole workflow in Slack.**

> Tutorial can be found [here](https://help.apify.com/en/articles/6454058-apify-integration-for-slack).

Reviewer: ```suggestion
A tutorial can be found [here](https://help.apify.com/en/articles/6454058-apify-integration-for-slack).
```

---

> Explore the [Slack integration tutorial](https://help.apify.com/en/articles/6454058-apify-integration-for-slack).

> [Slack](https://slack.com/) allows you to install various services in your workspace in order to automate and centralize jobs. Apify is one of these services, and it allows you to run your Apify actors, be notified about their run status, and get your results, all without opening your browser.

Reviewer: ```suggestion
[Slack](https://slack.com/) allows you to install various services in your workspace in order to automate and centralize jobs. Apify is one of these services, and it allows you to run your Apify Actors, get notified about their run statuses, and receive your results, all without opening your browser.
```
